src/gt4sd/domains/materials/scorer.py

```diff
@@ -352,6 +352,15 @@ def score(self, smiles: str) -> float:
         return Chem.QED.qed(Chem.MolFromSmiles(smiles))
 
 
+SCORING_FUNCTIONS = {
+    "rdkit_scorer": RDKitDescriptorScorer,
+    "tanimoto_scorer": TanimotoScorer,
+    "isomer_scorer": IsomerScorer,
+    "smarts_scorer": SMARTSScorer,
+    "qed_scorer": QEDScorer,
+}
```

Comment:
Maybe here we could add the remaining (more flexible) scorers, in order to have a singular place to access all the scorers

Reply:
Good point. I think I will move `SCORING_FUNCTIONS` in the init.

Reply:
If we add `PropertyPredictorScorer` we have problems with circular imports. Maybe we can solve these problems by relying more on `gt4sd.properties` in `domains`.